### ISSUE TITLE
C/C++ Addons - N-API

### ISSUE THREAD

**Issue by Globik** [paulhauner/example-async-node-addon#1]
What do you think about it?

**Comment by paulhauner** [comment]
Hey, sorry I didn't see this until just now!

I haven't been playing in Node/C++ for a long time so I am not aware of what these things are.

Do you want to implement them in this repo?

**Comment by Globik** [comment]
No, it's highly experimental, those feature. Already done in other repo. Nothing special, only an abi compatible  C code.
